projects/plugins/search/CHANGELOG.md

```diff
@@ -6,18 +6,27 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [1.1.0-beta] - 2022-07-29
-### Changed
-- E2E tests: bump @playwright/test and allure-playwright versions
-- Renaming master to trunk.
-- Renaming `master` references to `trunk`
-- Updated package dependencies.
+### Added
+- New Record Meter feature to show the breakdown of records in your search index.
 
 ### Fixed
-- E2E tests: fixed pretest cleanup script not running
-- Search: update plugin README
+- Don't open modal if only sort parameter is set.
```

Comment:
Should we include Github PR #s for these?

Would be helpful for us, but probably not important to our users.

Reply:
When running the changelogger tool the first time around,  you can pass `--add-pr-num` to add PR numbers, i.e. `jetpack release plugins/search changelog -a --add-pr-num`.

In this case and since you already have an existing changelog, you don't really need to worry about it I think.

Reply:
Thanks @jeherve. I'll leave those for now then. 

> jetpack release plugins/search

In our case we'd need to generate the changelog for the Search package I think? Not much goes on in the plugin itself.